RapidReact/src/main/java/frc/robot/subsystem/ClimberSubsystem.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystem;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import frc.robot.config.Config;
+import frc.robot.log.LogLevel;
+
+public class ClimberSubsystem extends BitBucketsSubsystem {
+
+  private WPI_TalonSRX climber;
+  private boolean enabledClimber;
+  private boolean disabledClimber;
+  private boolean fixedHook;
+  private boolean elevatorToggle;
+  private boolean elevatorExtend;
+  private boolean elevatorRetract;
```

Comment:
Let's remove these booleans. We will run the motor forwards/backwards to move the elevator up and down.